spec/features/users_can_view_projects_spec.rb

```diff
@@ -10,3 +10,16 @@
       expect(page).to have_content("19283746")
     end
 end
+
+RSpec.feature "Users can view a single project" do
+
+  scenario "by following a link from the home page" do
+    sign_in_with_user("user@education.gov.uk")
+    
+    Project.create!(urn: 19283746)
```

Comment:
It might be nice to assign the Project and then refer to the urn via that, for me it makes things that tiny bit more legible:
```ruby
single_project = Project.create!(urn: 19283746)
click_on single_project.urn
expect(page).to have_content(single_project.urn)
```